.github/CONTRIBUTING.md

```diff
@@ -44,7 +44,7 @@ your contributions.
 
 * All code must also pass [hlint](http://community.haskell.org/~ndm/hlint/) and [stylish-haskell](https://github.com/jaspervdj/stylish-haskell)
   with no warnings. This helps enforce a uniform style for all committers. Continuous integration will check this as well on every
-  pull request. There's a useful Makefile that helps with checking this locally. You can run `make commit-check` to do this manually but
+  pull request. There are useful tools in the nix-shell that help with checking this locally. You can run `NIX COMMAND HERE` to do this manually but
```

Comment:
Already talked to @monacoremo about that: A replacement for "make commit-check" / "make check" would be cool.

Reply:
Yes, it's going to be pretty simple to add a `postgrest-check` that checks everything. We could even run that one in CI, theoretically - but then we would lose some readability in the CI outputs, so best to keep the steps separate.

Reply:
Yeah, I agree that in CI the steps should be kept separate.